python/django-ninja/config.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+  website: http://django-ninja.rest-framework.com/
```

Comment:
could you remote `http` -> will be added automatically

Reply:
done